javadoc/allclasses-frame.html

```diff
@@ -1,29 +0,0 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
i removed Javadoc from the git repo, so here begins an enormous amount of deleted files